client/src/main/java/io/pravega/client/netty/impl/RawClient.java

```diff
@@ -151,4 +172,9 @@ public void close() {
         closeConnection(new ConnectionClosedException());
     }
 
+    @VisibleForTesting
+    public Long retreiveRequestId() {
```

Comment:
This shouldn't need to be here. The connection is being mocked, the mock can retrieve the IDs. This will avoid modifying the code.

Reply:
removed